src/workerd/io/io-context.c++

```diff
@@ -447,9 +448,11 @@ kj::Promise<void> IoContext::IncomingRequest::drain() {
     // For non-actor requests, apply the configured soft timeout, typically 30 seconds.
     timeoutPromise = context->limitEnforcer->limitDrain();
   }
+  context->pumpMessageLoop();
   return context->waitUntilTasks.onEmpty()
       .exclusiveJoin(kj::mv(timeoutPromise))
-      .exclusiveJoin(context->abortPromise.addBranch().then([] {}, [](kj::Exception&&) {}));
+      .exclusiveJoin(context->abortPromise.addBranch().then(
+          [] {}, [](kj::Exception&& e) { KJ_LOG(INFO, "uncaught exception", e); }));
```

Comment:
Is this added log intended? I'm afraid it will spam sentry

Reply:
It's an INFO log so it'll only show up in tests, but it looks unnecessary to me as any abort exception should have been reported elsewhere already, hence why it was being ignored here.

Reply:
Yup, this is only for tests. However I didn't see `Worker exceeded CPU time limit` log (if a FinalizationRegistry callback exceeds it) unless I added this log line here